spec/models/reviewable_user_spec.rb

```diff
@@ -243,5 +243,89 @@ def assert_require_reject_reason(id, expected)
       expect(event[:event_name]).to eq(:user_approved)
       expect(event[:params].first).to eq(user)
     end
+
+    describe "#scrub" do
+      it "scrubs the user history record" do
+        UserDestroyer.new(admin).destroy(user)
+        reviewable.reload
+
+        history = UserHistory.where(action: UserHistory.actions[:delete_user])
+        expect(history.count).to eq(1)
+        expect(history.first.details).to include("username: #{user.username}")
+        expect(history.first.ip_address).not_to be_blank
+
+        reviewable.scrub("reason", Guardian.new(admin))
+        reviewable.reload
+        history.reload
+
+        expect(history.first.details).to include("User details scrubbed by #{admin.username}")
+        expect(history.first.details).to include("reason")
+        expect(history.first.details).to include(
+          "Timestamp: #{Time.zone.parse(reviewable.payload["scrubbed_at"])}",
+        )
+
+        expect(history.first.details).not_to include(user.username)
+        expect(history.first.ip_address).to be_blank
+      end
+
+      it "doesn't scrub older user history records for the same username" do
+        # Create a history record with the same username but older than the reviewable
+        UserHistory.create!(
+          action: UserHistory.actions[:delete_user],
+          details: "id: #{user.id}\nusername: #{user.username}\nname: ",
+          created_at: 1.day.ago,
+          updated_at: 1.day.ago,
+          ip_address: "1.2.3.4",
+        )
+
+        UserDestroyer.new(admin).destroy(user)
+        reviewable.reload
+
+        history = UserHistory.where(action: UserHistory.actions[:delete_user])
```

Comment:
```suggestion
        history = UserHistory.where(action: UserHistory.actions[:delete_user]).order(created_at: :desc)
```

Minor suggestion to avoid potential test flakiness.